Eastern-Research-Group/mywaterway
# Feature/852 update dependencies

## Related Issues:
* [HMW-852](https://jira.epa.gov/browse/HMW-852)

## Main Changes:
* Updated to nodejs v22.
* Updated 3rd party dependencies.
* Updated ArcGIS JS API. During this go around I found that the root cause of the issue of printing the waterbody report was related to some css that was used to fix an issue with the layer list not being scrollable.

## Steps To Test:
1. Install nodejs v22.11.0
2. Navigate to http://localhost:3000/waterbody-report/MDE_EASP/MD-02140205/2022
3. Verify you can use the browser to print the page
4. Test out various other features
